plugins/discovery-ec2/src/main/java/org/elasticsearch/cloud/aws/Ec2Module.java

```diff
@@ -25,6 +25,6 @@
 
     @Override
     protected void configure() {
-        bind(AwsEc2Service.class).asEagerSingleton();
+        bind(AwsEc2ServiceImpl.class).asEagerSingleton();
```

Comment:
shouldn't this be bind(AwsEc2Service).to(AwsEc2ServiceImpl.class).asEagerSingleton()?